src/content/developers/docs/consensus-mechanisms/index.md

```diff
@@ -18,65 +18,75 @@ To better understand this page, we recommend you first read our [introduction to
+Nakamoto consensus is done by [miners](/developers/docs/consensus-mechanisms/pow/mining/) who compete to create new blocks full of processed transactions. They do this through a process known as "proof-of-work" mining (PoW). In PoW, miners are creating random numbers, called a nonce, and plugging that into a hash function to see if they produce a hash which satisfies a condition difficult enough to show they've put work into solving the problem. Once discovered, the miner who found the nonce proposes a new block to the rest of the network. In this new block, the miner pays itself in ETH. If the network decides to continue to build on his block, it becomes part of the "canonical chain" and the miner has some freshly minted ETH.
+
+The race is won by whosever computer can solve a math puzzle fastest – this produces the cryptographic link between the current block and the block that went before. Solving this puzzle puts the work in "proof-of-work."
 
 #### Security {#pow-security}
 
 The network is kept secure by the fact that you'd need 51% of the network's computing power to defraud the chain. This would require such huge investments in equipment and energy; you're likely to spend more than you'd gain.
 
 More on [proof-of-work](/developers/docs/consensus-mechanisms/pow/)
 
-### Proof-of-stake {#proof-of-stake}
+### Casper FFG {#casper-ffg}
 
 Ethereum has plans to upgrade to a **proof-of-stake (PoS)** consensus protocol.
 
 #### Block creation {#pos-block-creation}
 
-Proof-of-stake is done by validators who have staked ETH to participate in the system. A validator is chosen at random to create new blocks, share them with the network and earn rewards. Instead of needing to do intense computational work, you simply need to have staked your ETH in the network. This is what incentivises healthy network behaviour.
+**Casper the Friendly Finality Gadget (Casper FFG)** is utilized by validators who have staked ETH to participate in the system. A validator is chosen at random to create a new block. This randomness constitutes the proposal mechanism that would otherwise be the "work" in proof-of-work for Nakamoto consensus. Once it's a validator's turn to propose a block, they share it with the network and earn rewards. Instead of needing to do intense computational work, they'd simply need to have staked their ETH in the network. This is what incentivizes healthy network behavior as the validator now has skin in the game.
 
 #### Security {#pos-security}
 
-A proof-of-stake system is kept secure by the fact that you'd need 51% of the total staked ETH to defraud the chain. And that your stake is slashed for malicious behaviour.
+A proof-of-stake system is kept secure by the fact that you'd need 34% of the total voting power in a committee to defraud the chain. Malicious behavior under proof-of-stake also result in staked ETH being slashed in proportion to the size of the attack.
```

Comment:
Could you explain what you mean here by "defraud the chain?"

My understanding is a supermajority (67%) vote is required to justify and finalize epochs, thus if someone controlled 34% they could grind the network to a halt by preventing the chain from finalizing, but they would not be able to finalize their own malicious blocks on their own.

Reply:
Wow I totally forgot I did this. 

Honestly, I don't like that term "defraud the chain" ... so much so I question whether I wrote it in the first place LMAO! I probably did, but you're correct, you could halt... 

...or you could do what's known as "equivocation". If you give Coinbase one transaction and FTX another. Then you get to choose which epoch you like and go with it, censoring the partition of the network you don't want to execute. If you're in control over 34% you can do this to the network. I believe this is still possible. Double check with your lads internally. I'm just a guy who learned a thing or two in his brief stint at Ava Labs. I'm hardly a world-renowned expert.

Reply:
Let me rephrase that a bit... I said "Coinbase" and "FTX" but what I really should be saying is "32.5%" in both cases. You own (at least) 34%... You split the difference on the other two transactions by sending half one version of a transaction, another half the other version, and then wait until the epoch to see which version favors you the most.

Reply:
I don't think this can happen. 
 
Double-voting - i.e. voting for multiple targets for the same slot - is explicitly a slashable offence and because in this case it is backed by 33% of the total staked ether it would also trigger the inactivity leak. The inactivity leak would cause the clients on the minority fork to gradually bleed their stake, representing a smaller and smaller proportion of the total, eventually allowing the chain to begin finalizing again once a 66% supermajority is re-established. The malicious validators attempting a double-vote would get slashed and ejected from the network.

So a 33% stake is sufficient to halt the Beacon Chain from finalizing, but only temporarily.

Reply:
Yeah but the person responsible for slashing is the person with 34%. Slashing doesn't work. At all. We've been saying this.

Reply:
Oh and then the question is "well he wont be staked forever". Perhaps, but this person can censor any new staking transactions which usurps his control over the network as well. This is why at 34% attack is so incredibly damaging. Once that safety threshold is crossed, they have control over the entire network. All. Bets. Are. Off.

Reply:
No, sorry but that's not right. 

The person with 33% stake has no control over who is slashed. Slashing happens whenever a validator attests to multiple blocks for the same slot (there are other slashable offences too). This is detected by other validators on the network, and they get a small reward for pointing it out. The magnitude of the slashing would be especially high because a large number of validators are involved (correlation penalty) and the inactivity leak would gradually bleed their stake, re-establishing a supermajority attesting to the majority fork. 

A 33% stake does not allow you to choose who gets slashed.

Reply:
You really are not understanding this. Consensus is the process by which all databases are kept in-sync. The consensus protocol ensures that the slashing occurs. You can present all of the evidence that you want, but the person with 34% has control over making sure that the entire network comes to agreement on what the database (balances) look like. Individual nodes can update their databases to be in disagreement with the 34% node, but that doesn't actually ensure consensus. It's likely to throw those other nodes into an incorrect state causing them to get slashed.

Ok let me give you an example. Coinbase and FTX decide to collude against Binance. Together they have over 34% of the stake. They decide they're going to siphon money from Binance by presenting the network with two options: if number goes up half the network gets option A, if number goes down the other half gets option B. Then they attest positively to the option that benefits them the most and Binance the least. This double vote is only seen by the validators that they shared it to. Binance is completely cut out of the equation entirely. Their votes don't matter. Even if a double vote is detected, the other nodes do not have the power to ensure that a database update is replicated across all nodes. The double vote is meaningless. There is no mechanism design that can rectify this. 

All of that is meaningless though. The question of which vote is the correct vote ... there's only one vote. The 34% chose to cast ONE VOTE on the epoch. They cast a single vote on the correct state and the other 33% saw the wrong one but doesn't have the voting power to make a difference. 

Anyway. I've said all I can about this. It's immaterial, you've designed things as you want. I'll step out and wish y'all good luck.

Reply:
Thanks for the input @collincusce. I've asked some people working on pos consensus to weigh in here.